src/ducks/resource.js

```diff
@@ -74,6 +74,13 @@ function fetchTranslations(translated_id, type, project, language) {
         type: FETCH_TRANSLATIONS_SUCCESS,
         payload: { original, translations, loading: false }
       });
+    })
+    .catch((error) => {
+      console.warn('Unable to fetch translations', error);
```

Comment:
Why are you logging the error? Redux-logger is doing that for you.

Reply:
Redux-logger won't be there in production. I can take out the console warning, though. :man_shrugging: